cfgov/scripts/export_enforcement_actions.py

```diff
@@ -92,7 +82,7 @@ def export_actions(path='/tmp', http_response=False):
         write_questions_to_csv(response)
         return response
     file_path = '{}/{}'.format(path, slug).replace('//', '/')
-    with open(file_path, 'w', encoding='windows-1252') as f:
+    with open(file_path, 'w', encoding='utf-8') as f:
```

Comment:
When I look at the csv output in vim, it looks great. When I open it in Excel on a Mac, I see encoding weirdness on apostrophes and quotation marks, e.g. on [this page](http://localhost:8000/enforcement/actions/capital-one-bank/) (see the screenshot below). Not sure how it will appear to those opening in Excel on a Windows computer. Probably not an issue, but FYI, I guess.

<img width="550" alt="Screen Shot 2021-06-16 at 5 24 02 PM" src="https://user-images.githubusercontent.com/4295388/122295833-b24aea80-cec7-11eb-9023-501895415570.png">

Reply:
yeah I see the encoding weirdness as well. Spent a bit of time trying to get it fixed in excel for mac but eventually punted as I don't think it's widespread or important (and may not even be present on windows)